jsonschema/current/dungeoneer/ui/Hotbar.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "title": "Hotbar",
+    "type": "object",
+    "properties": {
+        "class": {
+            "const": "com.interrupt.dungeoneer.ui.Hotbar",
+            "description": "Fully-qualified Java class name."
+        },
+        "columns": {
+            "type": "integer",
+            "description": "The column count."
+        },
+        "rows": {
+            "type": "integer",
+            "description": "The row count."
+        },
+        "invOffset": {
+            "type": "integer",
+            "description": "TODO"
```

Comment:
I've noticed a few TODO descriptions here. If the fields aren't player facing, don't worry about documenting them.

Reply:
As discussed I will update these in a separate PR!